src/Nox.Solution/Extensions/EntityRelationshipExtensions.cs

```diff
@@ -46,7 +46,7 @@ public static bool HasRelationshipWithSingularEntity(this EntityRelationship rel
 
     public static bool ShouldUseRelationshipNameAsNavigation(this EntityRelationship relationship)
     {
-        var hasReferenceToSingularEntity = HasRelationshipWithSingularEntity(relationship);
+        var hasReferenceToSingularEntity = WithSingleEntity(relationship);
```

Comment:
I think bool starting with "With" is harder to understand, to be honest I like previous name more

Reply:
that is an extension method being called like a non extension, I will change to this `relationship.WithSingleEntity();` to have the desired usage